src/Language/PureScript/Make.hs

```diff
@@ -109,11 +117,19 @@ rebuildModuleWithIndex MakeActions{..} exEnv externs m@(Module _ _ moduleName _
     desugarCaseGuards elaborated
 
   regrouped <- createBindingGroups moduleName . collapseBindingGroups $ deguarded
+
   let mod' = Module ss coms moduleName regrouped exps
-      corefn = CF.moduleToCoreFn env' mod'
-      (optimized, nextVar'') = runSupply nextVar' $ CF.optimizeCoreFn corefn
+  traceM $ "PURUS START HERE: " <> T.unpack (runModuleName moduleName)
+  -- pTrace regrouped
+  -- pTrace exps
```

Comment:
Uncomment these and make traces visibility configurable by a global var or a flag